### DISCUSSION TITLE
Bulk selection lost after search

### DISCUSSION THREAD

**Discussion by ivanmar01** [rappasoft/laravel-livewire-tables#1722] [Q&A] [answered]
Using latest v3
Is there any way to preserve bulk selection after search?
I can get current array using $this->getSelected()
Any suggestions?

**Comment by lrljoe** [comment, marked as answer]
This was added here:
[https://github.com/rappasoft/laravel-livewire-tables/pull/1762](https://github.com/rappasoft/laravel-livewire-tables/pull/1762)
